addons/web/static/src/js/widgets/smooth_scroll.js

```diff
@@ -0,0 +1,315 @@
+odoo.define('web.smooth_scroll_on_drag', function (require) {
```

Comment:
This returns a class so we normally give the name of that class, so SmoothScrollOnDrag. Also the file was placed inside the widgets folder but this is not a widget... but I don't know... @aab-odoo could you tell us where to put this? :)

Reply:
Well the convention about JS module names changed recently (I expected @seb-odoo to communicate about it). It is now the path of the file, starting from module name: https://github.com/odoo/odoo/blob/master/addons/mail/static/src/widgets/discuss/discuss.js#L1

Reply:
For the initial question, I'd say in /core, even though I'm not convinced. But according to what it contains, and the other folders, it seems to be the appropriate place.

Reply:
JKE told me it was planned but I was indeed waiting for a communication about it. @IT-Ideas You can do as @aab-odoo says, sorry :wink:

Reply:
I didn't really get time to work on the doc yet. I just now created task-2293648 for this, we probably want to come with the final version together.